luma/lcd/device.py

```diff
@@ -323,6 +324,10 @@ class st7735(backlit_device):
     :type framebuffer: str
     :param bgr: Set to ``True`` if device pixels are BGR order (rather than RGB).
     :type bgr: bool
+    :param inverse: Set to ``True`` if device pixels are inversed.
+    :type inverse: bool
+    :param timed_reset: Set to ``True`` if device keeps white after init.
```

Comment:
Can the parameter be called ‘hard_reset’?

Reply:
Decided that the timed/hard reset should be implemented directly in the luma.core interface component instead, so i will raise a PR to address that directly in that repo instead